PhysicsTools/NanoAOD/python/nano_cff.py

```diff
@@ -47,6 +50,49 @@
    tauName=cms.string("Tau"),photonName=cms.string("Photon")
 )
 
+btagSFdir="/src/PhysicsTools/NanoAOD/data/btagSF/"
```

Comment:
this will not work with external data.
Use FileInPath

Reply:
argh I missed they wrote it that way... will fix